lib/devtools/config.rb

```diff
@@ -109,21 +109,16 @@ class Reek < self
 
     # Flay configuration
     #
-    # @note We cannot use unified Integer until 2.4 since `instance_of?(Integer)` is
-    #   not semantically equivalent to `instance_of?(Fixnum)`
-    #
-    # rubocop:disable Lint/UnifiedInteger
     class Flay < self
       FILE             = 'flay.yml'.freeze
       DEFAULT_LIB_DIRS = %w[lib].freeze
       DEFAULT_EXCLUDES = %w[].freeze
 
-      attribute :total_score, [Fixnum]
-      attribute :threshold,   [Fixnum]
+      attribute :total_score, [0.class]
```

Comment:
I think this is a good solution for 2.3 vs 2.4, once we drop < 2.4 we'll have to go to `Integer`.